pom.xml

```diff
@@ -27,6 +27,10 @@
   <artifactId>zendesk-java-client</artifactId>
   <version>0.2.5-SNAPSHOT</version>
 
+  <properties>
```

Comment:
properties are only for use when you reuse the dependency version in multiple places in the same pom or where you want to test multiple versions from the cli